includes/Core/Authentication/Authentication.php

```diff
@@ -373,6 +375,14 @@ function( $user ) {
 			}
 		);
 
+		add_filter(
+			'googlesitekit_features_request_data',
+			function( $body ) {
+				$body['connected_user_count'] = $this->count_connected_users();
```

Comment:
True, I agree that not having a separate filter will keep things together and simpler. However, in addition to needing a new `User_Options()` instance, I thought `count_connected_users()` sits better as a reusable public method within `Authentication` instead of in `Google_Proxy`. But I have moved the method into `GoogleProxy` for now.